common/config/azure-pipelines/templates/publish-test-results.yaml

```diff
@@ -16,34 +16,39 @@ steps:
       testRunTitle: "Clients - Context Registry Tests - $(Agent.OS) - ${{ parameters.NodeVersion }}"
       searchFolder: ${{ parameters.workingDir }}
     condition: succeededOrFailed()
+    enabled: false
```

Comment:
Can we delete/comment out these?

Reply:
I deleted most of the client ones and then just commented out the backend itwin client one.